operations/model.go

```diff
@@ -78,8 +80,10 @@ type ClientSettings struct {
 	UIServerHost          string              `json:"ui_server_host" yaml:"ui_server_host,omitempty"`
 	APIKey                string              `json:"api_key" yaml:"api_key,omitempty"`
 	User                  string              `json:"user" yaml:"user,omitempty"`
+	JWT                   string              `json:"jwt" yaml:"jwt,omitempty"`
 	UncommittedChanges    bool                `json:"patch_uncommitted_changes" yaml:"patch_uncommitted_changes,omitempty"`
 	AutoUpgradeCLI        bool                `json:"auto_upgrade_cli" yaml:"auto_upgrade_cli,omitempty"`
+	DoNotRunKanopyOIDC    bool                `json:"do_not_run_kanopy_oidc" yaml:"do_not_run_kanopy_oidc,omitempty"`
```

Comment:
For my own clarity, is the reason this is a "default on" flag because we'd like users to try the JWT first during the rollout and only set this in their evergreen.yml if they encounter issues?